aixplain/utils/file_utils.py

```diff
@@ -235,5 +237,42 @@ def s3_to_csv(s3_url: Text, aws_credentials: Optional[Dict[Text, Text]] = {"AWS_
         df.to_csv(output_path, index=False)
 
         return output_path
-    except (Exception, ValueError) as e:
+    except (Exception, ValueError) as e:  # ignore F841 # noqa
         raise Exception(e)
+
+
+def download_data_from_s3(
+    s3_path: Text,
+    local_filename: Text,
+    aws_credentials: Optional[Dict[Text, Text]] = {"AWS_ACCESS_KEY_ID": None, "AWS_SECRET_ACCESS_KEY": None},
```

Comment:
Regular use has no access to this credential. I would remove this method for now.